src/scenes/Activities/index.js

```diff
@@ -30,11 +30,47 @@ const messages = defineMessages({
 });
 
 class Activities extends React.Component {
+  static propTypes = {
+    intl: intlShape.isRequired, // eslint-disable-line react/no-typos
+    match: PropTypes.object.isRequired,
+    history: PropTypes.object.isRequired,
+    classes: PropTypes.object.isRequired,
+    actionableItemCount: PropTypes.object,
+  };
+
+  static defaultProps = {
+    actionableItemCount: undefined,
+  };
+
   constructor(props) {
     super(props);
 
+    // Determine tab index based on path
+    let tabIdx;
+    switch (this.props.match.path) {
+      case RouterPath.set: {
+        tabIdx = 0;
```

Comment:
Can we use a const to store the tab index value?

Reply:
done